types/react-date-range/index.d.ts

```diff
@@ -1,15 +1,15 @@
-// Type definitions for react-date-range 0.95
+// Type definitions for react-date-range 1.0
 // Project: https://github.com/Adphorus/react-date-range/
 // Definitions by: Junbong Lee <https://github.com/Junbong>
 //                 John Demetriou <https://github.com/DevsAnon>
+//                 Minseok Choi <https://github.com/Curzy>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.8
+// TypeScript Version: 4.0
```

Comment:
This file doesn't appear to use any features from 4.0 - this should be the *minimum TS version needed to support this file*, not the version you wrote the file with.

Reply:
Thank you for review. i returned TypeScript version to 2.8